src/app/item-page/bitstreams/request-a-copy/bitstream-request-a-copy-page.component.ts

```diff
@@ -42,18 +45,48 @@ export class BitstreamRequestACopyPageComponent implements OnInit, OnDestroy {
   bitstream$: Observable<Bitstream>;
   bitstream: Bitstream;
   bitstreamName: string;
+  form: UntypedFormGroup;
+  /**
+   * registration verification configuration
+   */
+  registrationVerification = false;
```

Comment:
This appears to be copied from the registration form.  Can we rename it?  This page has nothing to do with registration.  Maybe this needs to be named `requestCopyVerification`?